web/src/settings_data.ts

```diff
@@ -351,3 +351,10 @@ export function get_request_data_for_stream_privacy(selected_val: string): {
         }
     }
 }
+
+export function guests_can_access_all_other_users(): boolean {
+    const everyone_group = user_groups
+        .get_realm_user_groups_for_setting("can_access_all_users_group", "realm")
+        .find((group) => group.name === "role:everyone");
+    return realm.realm_can_access_all_users_group === (everyone_group?.id ?? false);
```

Comment:
A better way to do this would be `user_groups.get_user_group_from_id(group_id).name === "role:everyone"`.